src/helics/core/TimeCoordinator.hpp

```diff
@@ -100,6 +100,9 @@ class TimeCoordinator {
     bool disconnected{false};
     bool nonGranting{false};  // specify that the timeCoordinator should not grant times and
                               // instead operate in a continuous manner until completion
+    /// if set to true the time coordinator is joining an ongoing co-simulation
+    bool dynamicJoining{false};
```

Comment:
Mainly trying to avoid the actual variable and default value being split across two lines.  which was happening in a number of cases and making reading them a little more difficult